packages/wpcom-checkout/package.json

```diff
@@ -42,6 +42,7 @@
 		"@automattic/shopping-cart": "workspace:^",
 		"@emotion/styled": "^11.3.0",
 		"@stripe/stripe-js": "^1.53.0",
+		"@wordpress/compose": "^6.15.0",
```

Comment:
Are these intentional?

Reply:
They weren't, must've slipped when I was updating packages... Updated the PR